### PR TITLE
separate common-core and umbrella package configs

### PR DESCRIPTION
**Pull request by FedericoAmura**
---
name: 'common-core parameters not using all possible options from umbrella'
about: A new pull request
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/CODE_STYLE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description

This PR solves that when starting the common core package you are offered to pass an apiKey, which is not needed and should not be used on FE

Related issue: #`FILL_THIS_OUT`

### CODE REVIEW

File: packages/moralis/src/Config/MoralisConfig.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { ApiUtilsConfigValues, MoralisCoreConfigValues } from '@moralisweb3/common-core/src/index';
```
Comment (sogunshola) [inline packages/moralis/src/Config/MoralisConfig.ts:1]:
```suggestion
import { ApiUtilsConfigValues, MoralisCoreConfigValues } from '@moralisweb3/common-core';
```

File: packages/common/core/src/Config/MoralisConfig.ts [hunk 2]
```diff
@@ -4,33 +4,32 @@ import { EvmChainish } from './interfaces/EvmChainish';
...
+export interface EvmUtilsConfigValues {
   formatEvmAddress: EvmAddressFormat;
   formatEvmChainId: EvmChainIdFormat;
 }
 
 // @moralisweb3/api-utils
-interface ApiUtilsConfigValues {
+export interface ApiUtilsConfigValues {
   apiKey: string;
 }
```
Comment (b4rtaz) [inline packages/common/core/src/Config/MoralisConfig.ts:19]: Maybe this part should be moved to the `@moralisweb3/api-utils` package? If this is interface is not used in the `core` package, it shouldn't be there.

**Review by b4rtaz — CHANGES_REQUESTED** [review]

File: packages/moralis/src/Config/MoralisConfig.ts (hunk 1 above)
Comment (FedericoAmura) [reply packages/moralis/src/Config/MoralisConfig.ts]: Fixed

File: packages/common/core/src/Config/MoralisConfig.ts (hunk 2 above)
Comment (FedericoAmura) [reply packages/common/core/src/Config/MoralisConfig.ts]: You are right, missed it
Fixed

**Review by b4rtaz — APPROVED** [review]

**Review by sogunshola — APPROVED** [review]

**Pull request merged**